test/e2e/common/runtimeclass.go

```diff
@@ -84,6 +87,165 @@ var _ = ginkgo.Describe("[sig-node] RuntimeClass", func() {
 
 		expectPodRejection(f, e2enode.NewRuntimeClassPod(rcName))
 	})
+
+	/*
+		Release: v1.20
+		Testname: RuntimeClass API
+		Description:
+		The node.k8s.io API group MUST exist in the /apis discovery document.
+		The node.k8s.io/v1 API group/version MUST exist in the /apis/mode.k8s.io discovery document.
+		The runtimeclasses resource MUST exist in the /apis/node.k8s.io/v1 discovery document.
+		The runtimeclasses resource must support create, get, list, watch, update, patch, delete, and deletecollection.
+	*/
+	ginkgo.It(" [NodeConformance] should support RuntimeClasses API operations", func() {
+		// Setup
+		rcVersion := "v1"
+		rcClient := f.ClientSet.NodeV1().RuntimeClasses()
+
+		rc := runtimeclasstest.NewRuntimeClass(f.UniqueName+"-handler", f.UniqueName+"-conformance-runtime-class")
```

Comment:
as long as the handler is opaque to the conformance test and we do not drive any behavior implied to any specific handler, i am fine having this in conformance.  can we keep a note in the test that makes it clear that we should _never_ test a particular handler under conformance?

Reply:
added the comment

Reply:
this looks fine, but also note that there isn't any requirement that handlers be unique across runtime classes.